src/util.js

```diff
@@ -271,7 +271,7 @@ function timeif(bool, name, thunk) {
 }
 
 function pipeline(fns) {
-  return _.compose.apply(null, fns.reverse());
+  return _.flowRight.apply(null, fns.reverse());
```

Comment:
Would just `_.flow(fns)` be equivalent to this?

(I'm a little surprised to see that the current `pipeline` is mutating the `fns` argument, but I suppose `pipeline` is only used on arrays that are created for the sole purpose of being passed to it.)